src/components/new-request/search.vue

```diff
@@ -392,11 +426,12 @@ export default class Search extends Mixins(CommonMixin, NrAffiliationMixin) {
   get showJurisdiction (): boolean {
     // if (this.isAmalgamation) return true // *** FUTURE
     if (this.isNewXproBusiness) return true
+    if (this.isSelectedXproAndRestorable) return true
```

Comment:
New conditional whether to show the jurisdiction menu (when the business selected is restorable and an XPRO).